pyaedt/edb.py

```diff
@@ -1363,17 +1372,18 @@ def import_gds_file(
             have the same name. Only the extension differs.
         tech_file : str, optional
             Technology file. It uses Helic to convert tech file to xml and then imports the gds. Works on Linux only.
-
+        map_file : str, optional
+            Layer map file.
         Returns
```

Comment:
Do we need a blank line before the Returns section. I thought it was required?

Reply:
yep